meetings/2019-06-26.md

```diff
@@ -0,0 +1,69 @@
+# Node.js Foundation Technical Steering Committee (TSC) Meeting 2019-06-26
+
+## Links
+
+* **Recording**:  https://www.youtube.com/watch?v=B2DyWTJWk6c
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/722
+
+## Present
+
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Daniel Bevenius @danbev (TSC)
+* Gabriel Schulhof @gabrielschulhof (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Rich Trott @Trott (TSC)
+* Sam Roberts @sam-github (TSC)
+
+## Agenda
+
+### Announcements
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* http2: make compat finished match http/1 [#24347](https://github.com/nodejs/node/pull/24347)
+  * Anatoli is working reinstate his express test environment so that he can validate
+    if the issue he’d seen still exists or not, but it is slow going.
+
+### nodejs/Release
+
+* Development kit and Deployment kit releases [#341](https://github.com/nodejs/Release/issues/341)
+  * Discussion in the TSC meeting today was that we should postpone considering this further
+    until we have a list of things to either be removed from the smaller one or added to the larger
+    one before we move this forward.
+  * Removed from agenda until people add to lists required or otherwise want to add back
+    to the agenda.
+
+### nodejs/TSC
+
+* Nominating @BethGriggs to the TSC [#718](https://github.com/nodejs/TSC/issues/718)
+* doc: update TSC charter [#698](https://github.com/nodejs/TSC/pull/698)
+  * discussed today in the CPC meeting and it was agreed that it is ready to land.
+  * Michael has submitted PR to clarify CPC governance for review/approval of
+    charters so may wait until that lands to go ahead and land this.
+
+* Tracking issue for updating TSC on Board Meetings [#476](https://github.com/nodejs/TSC/issues/476)
+  * No board rep currently.  Vote is open for Node.js board rep as well as issues with
+    questions to the candidates.  Please review and make sure to vote.
+
+* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)
+  * N-API - Setting up meeting with WASM team, some interesting work from Ali and Ohad
```

Comment:
You mentioned there was a presentation about this. @mhdawson is it posted anywhere (a N-API meeting, maybe)?

Reply:
The link to the video is: https://youtu.be/vaN6zUWJH9w

Reply:
Thank you!